Setup/install.sh

```diff
@@ -81,6 +81,12 @@ install_dependencies() {
     sudo pip install -U RPi.GPIO
     sudo pip install pyusb
     sudo pip install numpy
+    sudo pip install python-periphery
+    sudo pip3 install -U RPi.GPIO
+    sudo pip3 install pyusb
+    sudo pip3 install numpy
+    sudo pip3 install python-periphery
```

Comment:
An Idea:  clone this repo to the DexterInd account.  Check on changes once per quarter.  This way, changes in the python-periphery library won't suddenly break things for us, forcing us to scramble and fix them.  

Just an idea, not a requirement.

Reply:
We're doing the fixed-version thing instead.